mobile/native/src/api.rs

```diff
@@ -58,14 +58,19 @@ pub struct Balances {
     pub lightning: u64,
 }
 
-/// Assembles the wallet info and publishes wallet info update event
+/// Assembles the wallet info and publishes wallet info update event.
 #[tokio::main(flavor = "current_thread")]
 pub async fn refresh_wallet_info() -> Result<()> {
     ln_dlc::refresh_wallet_info().await?;
 
     Ok(())
 }
 
+pub fn refresh_lightning_wallet() -> Result<()> {
```

Comment:
❓Why not the `#[tokio::main(flavor = "current_thread")]` syntax?

Reply:
Didn't you give me a review remark on the other PRs that you'd prefer using the runtime we keep static? 😅

Reply:
I think you misunderstood my point, see [this comment](https://github.com/get10101/10101/pull/1599/files#r1398761985).

But it's fair to consider what's the best thing to do here.

Using `#[tokio::main]` just expands to creating a _new_ runtime and blocking on it. Alternatively we could use the "dedicated backend runtime" like you do here. [In the past](https://github.com/get10101/10101/blob/4d4a9a3e857c3bd3d8e49a7164826f79e4050424/mobile/native/src/ln_dlc/mod.rs#L128) we did run into problems because we were doing heavy work in the newly created runtime and that could block the UI thread. Maybe what you did here could make sense.

In any case, if you make `refresh_lightning_wallet` sync like it should be you won't need a runtime anyway.